src/HttpClientConfigurator.php

```diff
@@ -98,4 +101,28 @@ public function setApiKey(string $apiKey)
 
         return $this;
     }
+
+    /**
+     * @param Plugin $plugin
+     *
+     * @return HttpClientConfigurator
+     */
+    public function appendPlugin(Plugin $plugin)
```

Comment:
The way you would call the method can be the same, but you can also pass multiple plugins in one call if you want.